controllers/apply/form-router/lib/salesforce.js

```diff
@@ -1,78 +1,69 @@
 'use strict';
-const request = require('request-promise-native');
+const got = require('got');
+const FormData = require('form-data');
 const { SALESFORCE_AUTH } = require('../../../../common/secrets');
 
 class Salesforce {
     constructor(apiUrl, accessToken) {
-        this.apiUrl = apiUrl;
         this.apiVersion = 'v45.0';
-        this.headers = { Authorization: `Bearer ${accessToken}` };
-    }
-    get(urlPath) {
-        return request.get({
-            uri: `${this.apiUrl}${urlPath}`,
-            headers: this.headers,
-            json: true,
-        });
-    }
-    postJson(urlPath, body) {
```

Comment:
Inlined this as using the got instance directly is now a similar amount of code to using this abstraction, and basically doing the same thing but at a lower level.